bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeBToTH_Hbb_LH/TprimeBToTH_Hbb_LH_MT600_MH75/TprimeBToTH_Hbb_LH_MT600_MH75_run_card.dat

```diff
@@ -0,0 +1,267 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+tag_1     = run_tag ! name of the run
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+100000 = nevents ! Number of unweighted events requested
```

Comment:
This many events for gridpack pilot run will only create some lags. Suggest to reduce it to 2-3K